x-pack/plugins/canvas/public/state/selectors/__tests__/workpad.js

```diff
@@ -128,7 +128,6 @@ describe('workpad selectors', () => {
       expect(selector.getSelectedPage({})).to.be(undefined);
       expect(selector.getPageById({}, 'page-1')).to.be(undefined);
       expect(selector.getSelectedElement({})).to.be(undefined);
-      expect(selector.getSelectedElementId({})).to.be(undefined);
```

Comment:
CR: This test should have been removed, just updated to check for `null` (or falsy).

Reply:
You meant to say, should _not_ have been removed? Due to having phased out all its uses except as implementation detail (mostly, to `getSelectedElement` which is still tested), it is no longer exported, and as such we don't even have access to it here. I'd prefer not exporting it - even its two callers are deprecated themselves.

Reply:
Yes I did mean *should not*. But I missed that only the other selector uses it now. Maybe we can remove the export to avoid its use in the meantime then?

Reply:
Thanks, the 🐐📰 is that it's already not exported :-) see comment above

Reply:
Indeed it's not, my bad.